### PR TITLE
Add 2022q2 clusteradm report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

**Comment by lwhsu** [comment]
Please wait at least one day for other clusteradm members to comment.
(I know there is a "draft" state but it seems I cannot set it or I didn't find the right way to do so.)

File: 2022q2/clusteradm.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* Plan Hardware refresh, and fixing misc failures in each sites
+* Improve the package building infrastructure
+* Review the service jails and service administrators operation
+* Working with doceng@ to improve https://www.freebsd.org and https://docs.freebsd.org
+* Improve the web service architecture
+* Improve the cluster backup plan
+* Improve the log analysis system
+
+We are in particular searching for possible hosting providers in Europe for setting up a new mirror.
+Please contact us if there is any possibilities for fitting the requirements for a link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] or a link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny mirror].
```
Comment (ppaeps) [inline 2022q2/clusteradm.adoc:27]: I would suggest rewriting this paragraph:


We are looking for an additional full mirror site (five servers) in Europe.  See link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] for our needs.  Offers of additional single-server mirrors (see link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny mirror]) are always welcome too, especially in Europe.

File: 2022q2/clusteradm.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[https://www.freebsd.org/administration/#t-clusteradm]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronise its distributed work and communications.
+In this quarter, the team has worked on the following:
+
+* Installed a new mirror in Sydney, Australia hosted by IX Australia
+* Fixed CI cluster hardware failure
+* Set up a new internal monitoring system
```
Comment (ppaeps) [inline 2022q2/clusteradm.adoc:13]: * Regular cluster-wide software upgrades

**Review by ppaeps — CHANGES_REQUESTED** [review]
Minor additions and improvements.

**Comment by lwhsu** [comment]
> Minor additions and improvements.

Thanks and updated.

**Review by ppaeps — APPROVED** [review]
Looks good.  Thank you!

**Comment by pauamma** [comment]
> Please wait at least one day for other clusteradm members to comment. (I know there is a "draft" state but it seems I cannot set it or I didn't find the right way to do so.)

Do you see "Still in progress? Convert to draft" below "Reviewers" in the right sidepane? If you do, click on "Convert to draft".

Also, could you avoid force-pushing next time? See item 5 under "Please submit your draft as early as possible. To submit:" in README.adoc.

File: 2022q2/clusteradm.adoc [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+* Regular cluster-wide software upgrades
+* Regular support for FreeBSD.org user accounts
+
+Work in progress:
+
+* Work with the PowerPC team to improve the package builders, universal, and reference machines.
+* Plan Hardware refresh, and fixing misc failures in each sites
+* Improve the package building infrastructure
+* Review the service jails and service administrators operation
+* Working with doceng@ to improve https://www.freebsd.org and https://docs.freebsd.org
```
Comment (pauamma) [inline 2022q2/clusteradm.adoc:23]: Clarify: improve contents? Improve hosting performance? Something else?

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q2/clusteradm.adoc (hunk 3 above)
Comment (ppaeps) [reply 2022q2/clusteradm.adoc:23]: Improvements to the continuous deployment infrastructure driving the publicly visible builds of www.f.o and docs.f.o.

Comment (pauamma) [reply 2022q2/clusteradm.adoc:23]: > Improvements to the continuous deployment infrastructure driving the publicly visible builds of [www.f.o](http://www.f.o) and docs.f.o.

Thanks. In the interest of not delaying the status reports, do you mind if I merge and s/improve/improve deployment of/?

Comment (ppaeps) [reply 2022q2/clusteradm.adoc:23]: Please do.  Thank you!

**Pull request merged**